README.md

```diff
@@ -15,13 +15,9 @@ Speakerfight is an arena where speakers can fight ~against~ each other and peopl
 $ git clone git@github.com:luanfonceca/speakerfight.git
 $ cd speakerfight
 $ pip install -r requirements.txt
```

Comment:
What about a `make deps` to install this? Or "deps" looks like another kind of dependencies, not only python deps i mean.